cfgov/v1/models/base.py

```diff
@@ -539,3 +539,9 @@ class TemporaryLockout(models.Model):
     user = models.ForeignKey(User)
     created = models.DateTimeField(auto_now_add=True)
     expires_at = models.DateTimeField()
+
+
+class Feedback(models.Model):
+    is_helpful = models.BooleanField()
+    comment = models.TextField(blank=True, null=True)
+    page = models.ForeignKey(Page, on_delete=models.SET_NULL, null=True)
```

Comment:
The association as a foreign key makes more sense to me than a page URL, especially considering the possibility for a slug/structure change we'd have to account for.

I'll most definitely create a `related_name` though.